portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -15,6 +15,8 @@
 package com.google.sps.servlets;
 
 import java.io.IOException;
+import com.google.gson.Gson;
```

Comment:
nit: alphabetizing the imports can help make it easier to find a relevant import later and make sure you're not adding the same one twice

Reply:
Nice tip never thought about that!